homeassistant/components/rflink/__init__.py

```diff
@@ -3,9 +3,10 @@
 from collections import defaultdict
 import logging
 import async_timeout
-
 import voluptuous as vol
 
+from rflink.protocol import create_rflink_connection
+from serial import SerialException
 from homeassistant.const import (
```

Comment:
Please add a blank line between 3rd party and homeassistant import groups.

If you use the isort utility it will sort and format imports automatically.

Reply:
Thanks for pointing me to the _isort_ utility. Done.
I have also applied the _isort_ to the test classes.